styles.css

```diff
@@ -321,10 +321,6 @@ a:link {
   flex-wrap: wrap;
 }
 
```

Comment:
### WORKS SECTION


- Hey, your work's section seems to overlap with the image in the first section. please, kindly improve the design. see the image below:


*your design*
![Screenshot from 2023-03-30 06-12-20](https://user-images.githubusercontent.com/59397159/228737458-1ae4cbdf-c2af-4d43-b33b-e7d64afba86f.png)

*Figma design*
![Screenshot from 2023-03-30 06-25-44](https://user-images.githubusercontent.com/59397159/228737891-cfbb17de-8b0d-4d4c-a24f-3a15f22b8358.png)

Reply:
### ABOUT SECTION 


- Hey, also, there should be space between the paragraph and the background image. see the image below to improve the design of this section:


*your design*
![Screenshot from 2023-03-30 06-13-59](https://user-images.githubusercontent.com/59397159/228737618-023bdfb3-cc0a-409e-bbc3-01e714c6f294.png)


*Figma design*
![Screenshot from 2023-03-30 06-27-55](https://user-images.githubusercontent.com/59397159/228738567-11dcdb83-72c0-4fec-9b8a-c18dcff2f4bb.png)

Reply:
### ABOUT LANGUAGE LIST


- Hey, let's also improve the design on this card as the contents are coming out of the card. see the image below to improve your design:

*Figma design*
![Screenshot from 2023-03-30 06-33-48](https://user-images.githubusercontent.com/59397159/228739427-e9fd05e0-4b4b-472d-9ace-27508800ee8c.png)

*Your design*
![Screenshot from 2023-03-30 06-33-28](https://user-images.githubusercontent.com/59397159/228739429-99f99f30-0b4a-4b48-8566-8b71a1d835b2.png)
*your design*
![Screenshot from 2023-03-30 06-38-20](https://user-images.githubusercontent.com/59397159/228739788-25f26bc1-bab4-4921-a186-b8e22a782e14.png)